applications/maplab-node/src/map-builder-flow.cc

```diff
@@ -87,6 +88,12 @@ MapBuilderFlow::MapBuilderFlow(
           external_resource_folder_);
     }
   }
+
+  if (sensor_manager.hasSensorOfType(vi_map::SensorType::kExternalFeatures)) {
+    stream_map_builder_.setExternalFeaturesSyncToleranceNs(
+        FLAGS_vio_nframe_sync_tolerance_ns);
```

Comment:
I think we should introduce a new fläg for this tolerance instead of reusing `FLAGS_vio_nframe_sync_tolerance_ns`. The odometry might be coming in from a different sensor and thus, the delta to the next pose can be quite large in such cases.